README.md

```diff
@@ -98,7 +98,7 @@ running multiple schedulers -- please see the "Scheduler" docs.
 
 Visit the official Airflow website documentation (latest **stable** release) for help with [installing Airflow](https://airflow.apache.org/installation.html), [getting started](https://airflow.apache.org/start.html), or walking through a more complete [tutorial](https://airflow.apache.org/tutorial.html).
 
-> Note: If you're looking for documentation for master branch (latest development branch): you can find it on [ReadTheDocs](https://airflow.readthedocs.io/en/latest/).
+> Note: If you're looking for documentation for master branch (latest development branch): you can find it on [s.apache.org/airflow-docs](https://s.apache.org/airflow-docs/).
```

Comment:
Probably that was discussed somewhere, but why we cannot have latests docs at `airflow.apache.org`?

Reply:
For four reasons:
1. airflow.apache.org is deployed using Git and therefore rolling in new versions as regularly as it would be problematic right now. The size of the repository would explode quickly
2. Only committers or Github Action in the same repository can push new commit to airflow-site.
3. This is a development version and we should not publish it on the official website as this could encourage users to install unofficial released versions. Unofficial versions can only be distributed by mailing list and the like.
4. The website is not yet prepared for the new documentation structure.  For now, we can only publish documentation for one apache-airflow package. Soon I will be working on updating the theme to support multiple packages.

> **Do not include any links on the project website that might encourage non-developers to download and use nightly builds, snapshots, release candidates, or any other similar package.**

http://www.apache.org/legal/release-policy.html